packages/paste-core/components/file-picker/src/index.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Label, RequiredDot} from '@twilio-paste/label';
+import {Text} from '@twilio-paste/text';
+import {useUID} from '@twilio-paste/uid-library';
+import {Button} from '@twilio-paste/button';
+import {MediaObject, MediaBody, MediaFigure} from '@twilio-paste/media-object';
+import {Box} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+export interface FilePickerProps extends React.HTMLAttributes<HTMLInputElement>, Pick<BoxProps, 'element'> {
+  disabled?: boolean;
+  required?: boolean;
+}
+
+const FilePicker = React.forwardRef<HTMLInputElement, FilePickerProps>(
+  ({element = 'FILEPICKER_CONTAINER', disabled = false, required = false, ...props}, ref) => {
+    const [fileName, setFileName] = React.useState('No files selected');
+
+    const id = useUID();
+
+    const handleChange = (evt: any): void => {
+      const file = evt.target.value;
+      setFileName(file);
+      console.log('the event', evt);
+      console.log('file name', fileName);
+    };
+
+    return (
+      <MediaObject {...props}>
+        <MediaFigure spacing="space30">
+          <Label htmlFor={id} element={element}>
```

Comment:
Suggestion: Due to the nature of the component, and Label, I suspect we might be better off not using `Label` internally. I don't think you need any of the styles, and if we update the styles to Label, that might have unintended concequences on File Picker. It should be a label element, maybe just a Box as label.

```
<Label>File picker label</Label>
<FilePicker />
<HelpText>help text goes here underneath</HelpText>